rosdep/base.yaml

```
@@ -8487,6 +8487,15 @@ rustc:
   opensuse: [rust]
   rhel: [rust]
   ubuntu: [rustc]
+rustup:
+  alpine: [rustup]
+  arch: [rustup]
+  debian: [rustup]
+  fedora: [rustup]
+  gentoo: [dev-util/rustup]
+  nixos: [rustup]
+  opensuse: [rustup]
+  ubuntu: [rustup]
```

You can make it version-specific, e.g.:

```suggestion
  ubuntu: [rustup]
    '*': [rustup]
    focal: null
    jammy: null
```